### DISCUSSION TITLE
Hyper sonic

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#75] [Q&A] [answered]
Do i make hyper sonic for my game

**Comment by Techokami** [comment, marked as answer]
The Q&A category is for asking questions about the framework, not for discussing your projects. Why not make a thread in the Show and Tell category and show us what you have planned, and keep design questions for your project in that?

**Comment by DimensionWarped** [comment]
Just going to add, if you are going to have a show and tell thread for your project, please keep it to a single thread.

**Reply by Raptorq1** [reply]
Sorry what I was trying to say is how do I add hyper sonic to my project.
